src/cryoemservices/wrappers/clem_align_and_merge.py

```diff
@@ -349,6 +448,7 @@ class AlignAndMergeParameters(BaseModel):
     series_name: str
     images: Path | list[Path]
     metadata: Optional[Path] = Field(default=None)
+    crop_to_n_frames: Optional[int] = Field(default=None)
```

Comment:
Allowing None here might not be the best solution. Could you take an int value (perhaps 0 or -1) to mean "don't do this"?